libs/service-portal/activities/src/module.ts

```diff
@@ -0,0 +1,29 @@
+import { lazy } from 'react'
+
+import { ApiScope } from '@island.is/auth/scopes'
+import { PortalModule, PortalRoute } from '@island.is/portals/core'
+import { Features } from '@island.is/react/feature-flags'
+
+import { m } from './lib/messages'
+import { ActivitiesPaths } from './lib/paths'
+
+const allowedScopes: string[] = [ApiScope.internal, ApiScope.internalProcuring]
+
+export const activitiesModule: PortalModule = {
+  name: m.activities,
+  featureFlag: Features.servicePortalActivitiesModule,
+  enabled({ userInfo }) {
+    return userInfo.scopes.some((scope) => allowedScopes.includes(scope))
+  },
+  routes() {
+    const routes: PortalRoute[] = [
```

Comment:
nitnit: You can just return the array without defining the type since routes return type is PortalRoute[]